cmd/goQuery/commands/root.go

```diff
@@ -108,22 +117,36 @@ func initLogger() {
 	}
 }
 
+// initConfig reads in config file and ENV variables if set. goQuery doesn't need one to run
+// as a CLI tool. The functionality exists to set some defaults for e.g. the query-server
+func initConfig() {
+	if cfgFile != "" {
+		// Use config file from the flag.
+		viper.SetConfigFile(cfgFile)
+
+		viper.AutomaticEnv() // read in environment variables that match
+
+		// If a config file is found, read it in.
+		if err := viper.ReadInConfig(); err != nil {
+			fmt.Fprintf(os.Stderr, "Failed to read in config: %v\n", err)
```

Comment:
It could, but we shouldn't call init logger before getting the configuration because that - you guessed it - possibly contains info on how to init the logger.

I propose something more rudimentary:
```
logger := logging.New(logging.LevelInfo, logging.EncodingPlain, logging.WithOutput(os.Stderr))
```
and using that. Means that we need to code the `Plain` encoder, but that shouldn't be too much of a hassle.

Reply:
Yeah, fair point, or maybe even providing a default / global plain logger for use from everywhere (that gets initialized with `init()`, like most other logging packages provide)?